extensions/dart/languages/dart/highlights.scm

```diff
@@ -210,17 +213,14 @@
 ] @keyword.coroutine
 
 [
- (const_builtin)
- (final_builtin)
  "abstract"
-  "covariant"
+ "covariant"
  "dynamic"
  "external"
  "static"
- "final"
-  "base"
-  "sealed"
- ] @type.qualifier
+ "base"
+ "sealed"
+] @type.qualifier
```

Comment:
I think all of these keywords should be `keyword`, instead of `type`, since they are not type names. And I would include `final` and `const` as keywords as well (not constants).

 Thoughts?